dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
Forgive my java packaging ignorance, but why is this file in this repo? Isn't it in a separate metric utils java repo?

Reply:
This is the shim between the OpenTelemetry SDK and our library, transforming the OpenTelemetry data types for use with the metric utils API. It seems line the name is not ideal - probably something like `MetricAdapter` would be more clear?

Reply:
Yeah the name confused me a little. I saw the name and just assumed it was a file that you forgot to delete.